index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Setup-and-Mobile-Version-skeleton</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="header">
+      <a href="#"><h4>Welcome</h4></a>
```

Comment:
[ Optional Changes ] This change won't stop your project from being approved but will help improve it. 
- [ ] It would be nice to use your name in the logo section. The idea here is that this project will serve as your personal portfolio website and you're doing great designing it. The Welcome text is meant to be a placeholder for your logo or name. 👍.